src/System.Private.CoreLib/src/System/Diagnostics/Debugger.cs

```diff
@@ -6,11 +6,14 @@
 // and is used for communicating with a debugger.
 
 using System.Runtime.CompilerServices;
+using System.Threading;
 
 namespace System.Diagnostics
 {
     public static class Debugger
     {
+        private static bool s_triggerTAEForDebugger;
```

Comment:
Could you please use the full name `s_triggerThreadAbortExceptionForDebugger` ?

This is de-facto public API. We not use cryptic TLAs in named of public APIs.

Reply:
The latest change does not have this field anymore.